Tasks/AzureResourceGroupDeployment/operations/ResourceGroup.ts

```diff
@@ -219,7 +220,7 @@ export class ResourceGroup {
         var template: Object;
         try {
             tl.debug("Loading CSM Template File.. " + this.taskParameters.csmFile);
-            template = JSON.parse(fileEncoding.readFileContentsAsText(this.taskParameters.csmFile));
+            template = JSON.parse(stripJsonComments(fileEncoding.readFileContentsAsText(this.taskParameters.csmFile)));
```

Comment:
Do you have any L0 tests? if there are existing tests, you can update them to have comments.